BlazorAccessibilityTests.Tests/AriaAccordionTests.razor

```diff
@@ -1,335 +1,197 @@
 ﻿@using BlazorAccessibilityTests.Shared
 @inherits TestContext
 @code {
-
-    // Element that serves as an accordion header.
-    // Each accordion header element contains a button that controls the visibility of its content panel.
-    // The example uses heading level 3 so it fits correctly within the outline of the page; the example is contained in a section titled with a level 2 heading.
-    [Fact(DisplayName = "Accordion header is rendered.")]
-    public void HeaderElementShouldRender()
+    private IRenderedFragment RenderAccordionWithTwoPanels() 
+        => Render(@<AriaAccordion>
+                     <AccordionPanel Title="P1">
+                       <p>P1 Content</p>
+                     </AccordionPanel>
+                     <AccordionPanel Title="P2">
+                       <p>P2 Content</p>
+                     </AccordionPanel>
+                   </AriaAccordion>);
+    
+    private IRenderedFragment RenderAccordionWithFourPanels() 
+        => Render(@<AriaAccordion>
+                     <AccordionPanel Title="P1">
+                       <p>P1 Content</p>
+                     </AccordionPanel>
+                     <AccordionPanel Title="P2">
+                       <p>P2 Content</p>
+                     </AccordionPanel>
+                     <AccordionPanel Title="P3">
+                       <p>P3 Content</p>
+                     </AccordionPanel>
+                     <AccordionPanel Title="P4">
+                       <p>P4 Content</p>
+                     </AccordionPanel>
+                   </AriaAccordion>);
+
+    [Fact]
+    public void FirstRenderMarkupCorrect()
```

Comment:
This test just verifies that the basic markup is correct.

The `MarkupMatches` method uses the `:regex` modifier in the markup to validate that there is an attribute with some content inside it.

Then, the following tests will verify that the relationship between the attributes (labelled by, controlled by) is correct.